### ISSUE TITLE
Multi Cursor stopped working for me today

### ISSUE THREAD

**Issue by Pendemonium** [joseramonc/multi-cursor#50]
I'm using Atom v1.48.0 x64, and multi-cursor 3.0.0. 

I have automatic updates turned on.

The packages I have installed are:

- atom-wrap-in-tag
- auto-update-plus
- browse
- build
- busy-signal
- ctrl-dir-scroll
- double-tag
- minimap
- multi-cursor
- satisfy-dependencies
- set-syntax
- sublime
- sublime-block-comment
- Sublime-Style-Column-Selection
- sublime-word-navigation
- sublimify
- tag
- teletype
- text-manipulation

---

Hopefully this gets sorted out soon. I never realized how much I would miss it, and how much my workflow benefitted from it, until it stopped working.

**Comment by joseramonc** [comment]
Hey! Is any error popping up? Could you confirm if the command is also failing via the command palette (<kbd>cmd</kbd> + <kbd>shift</kbd> + <kbd>P</kbd> and then choose `Multi Cursor: Expand Up` or similar)

**Comment by Pendemonium** [comment]
I'm not seeing an error, and I followed your directions with no positive results:

<img width="645" alt="Screen Shot 2020-06-24 at 1 33 31 PM" src="https://user-images.githubusercontent.com/4502146/85619359-6ac8b000-b61f-11ea-9e5f-eb8cd82bc92c.png">

"Expand All Up" did something:

<img width="349" alt="Screen Shot 2020-06-24 at 1 35 43 PM" src="https://user-images.githubusercontent.com/4502146/85619502-a19ec600-b61f-11ea-8e87-f01626c1de12.png">

But command-clicking on the lines I want to multi-select is not working.

I've restarted Atom twice. When I have time I'll try restarting my computer.
